docs/asset_location_id.md

```diff
@@ -1,7 +1,7 @@
 
 ## Asset `location_id` quick reference
 
-- `item_id` of the parent asset (Used to build the assets tree, see example below)
+- `location_id` of the parent asset (Used to build the assets tree, see example below)
```

Comment:
This is supposed to be `item_id`: the `location_id` of the child asset is the `item_id` of the parent asset.

Reply:
sure, that makes sense if you think about it that way. in my head i just thought about that when considering the return of asset endpoints location_id refers to the parent